data/lm/generate_package.py

```diff
@@ -1,17 +1,11 @@
 #!/usr/bin/env python
 from __future__ import absolute_import, division, print_function
 
-# Make sure we can import stuff from util/
-# This script needs to be run from the root of the DeepSpeech repository
-import os
-import sys
-
-sys.path.insert(1, os.path.join(sys.path[0], "..", ".."))
-
 import argparse
 import shutil
+import sys
```

Comment:
I don't think we need it anymore (likely to be true at a lot of other places)

Reply:
It's used in the script.

Reply:
Is it? I searched and fail to find it, but if you say so :)

Reply:
https://github.com/mozilla/DeepSpeech/blob/a05baa35c96a698d5b7836a4351968bf6dda2734/data/lm/generate_package.py#L51